.travis.yml

```diff
@@ -1,7 +1,15 @@
-sudo: false
-language: node_js
+sudo: required
+language: python
+python: 3.6
+
+before_install:
+  - nvm install 8
+  - npm install
+  - npm install -g eslint prettier eslint-plugin-prettier
```

Comment:
Why?

Reply:
https://travis-ci.org/coala/gci-leaders/builds/312179396#L748

Reply:
you can add `node_modules/.bin` to PATH
and set NODE_PATH (see our dockerfile)

https://github.com/coala/docker-coala-base/blob/master/Dockerfile#L10
https://github.com/coala/coala-atom/blob/master/circle.yml#L32